AlphaWallet/Protection/Coordinators/SplashCoordinator.swift

```diff
@@ -9,13 +9,16 @@ class SplashCoordinator: Coordinator {
 
     init(window: UIWindow) {
         self.window = window
+        window.backgroundColor =  .white;
```

Comment:
What is this for?

Reply:
This was making background color of splash view controller white as on app launch when its loaded on window object it was showing black  screen with logo after wallet creation